### ISSUE TITLE
<podcast:person> loose ends?

### ISSUE THREAD

**Issue by daveajones** [Podcastindex-org/podcast-namespace#96]
Are we comfortable with this format to talk about finalizing, or are there any loose ends that need to be hammered out?

```
<podcast:person role="[host or guest]" name="[name of person]" img="[(uri of content)]" href="[(uri to website/wiki/blog)]" />
```

**Comment by normand1** [comment]
LGTM!

**Comment by brianoflondon** [comment]
Are there going to be guidelines for the image size/shape and or the bio format?

**Comment by cisene** [comment]
Is there bits/pieces that can be borrowed from the FOAF (Friend-o-a-Friend) specification (http://xmlns.com/foaf/spec/)?

It's to describe persons/people and what relation they have.

**Comment by daveajones** [comment]
> 
> 
> Are there going to be guidelines for the image size/shape and or the bio format?

Definitely image size/shape.  Bio format seems like just a link for now.  Wikipedia, IMDB, etc.

**Comment by snookfin** [comment]
Yes. As written.

### Suggestion for Images (following Apple's artwork specs):
**Size:** square; minimum 600 x 600 pixels and maximum 1200 x 1200 pixels (preferred)
**Resolution:** 72 dpi
**File type:** JPEG or PNG
**Colorspace:** RGB

**Comment by daveajones** [comment]
Forgot to mention, also Podchaser credits.

**Comment by snookfin** [comment]
Actually, I'm rethinking this a bit. I may actually be in favor of leaving it the way it's currently documented on the [README.md](https://github.com/Podcastindex-org/podcast-namespace/blob/main/README.md).

`<podcast:person role="[host or guest]" img="[(uri of content)]" href="[(uri to website/wiki/blog)]">[name of person]</podcast:person>`

The only reason I'm thinking this may be better is that the name of the person is actually content; something a human would read and understand.

At the end of the day though, this is not a big deal. I'm fine with @daveajones just picking one so we can build it out.

**Comment by daveajones** [comment]
I don't care either. Let's keep it the way it is then.  I agree with that sentiment.

Let this fester for a few more days and if nothing else comes up we can go ahead and set it in stone?

**Comment by bslinger** [comment]
It would be valuable to allow for multiple links to identifiying URIs, be they social media accounts, a Wikipedia page or a Podchaser Creator Profile - this would allow for the consuming platform to use whichever verification method they feel comfortable with to disambiguate people, as well as providing more options for linking out to useful pages for a specific person.

It's not quite as neat, but something like:

```
<podcast:person role="[host or guest]" img="[(uri of content)]">
  <podcast:personName>[name of person]</podcast:personName>
  <podcast:personUri>https://twitter.com/[username]</podcast:personUri>
  <podcast:personUri>https://wikipedia.org/[page name]</podcast:personUri>
  <podcast:personUri>https://podchaser.com/c/[Podchaser ID]</podcast/personUri>
</podcast:person>
```

We could probably remove the `personXX` prefix as well if we're happy to rely on the context of the parent element to know what it relates to.

**Comment by bslinger** [comment]
Here are the roles put together by the Podcast Taxonomy Consortium as well - it's still getting a few edits but it's mostly complete. I'm not sure if this belongs in the spec or not, but it was put together so that we have a known set of possible roles for credits on a podcast.
[Podcast Taxonomy Consortium 2020.zip](https://github.com/Podcastindex-org/podcast-namespace/files/5484830/Podcast.Taxonomy.Consortium.2020.zip)

**Comment by daveajones** [comment]
> Here are the roles put together by the Podcast Taxonomy Consortium as well - it's still getting a few edits but it's mostly complete. I'm not sure if this belongs in the spec or not, but it was put together so that we have a known set of possible roles for credits on a podcast.
> [Podcast Taxonomy Consortium 2020.zip](https://github.com/Podcastindex-org/podcast-namespace/files/5484830/Podcast.Taxonomy.Consortium.2020.zip)

This is so extensive.  It seems like it would belong in it's own element consideration.  Like <podcast:credits> with a link to an extensive credits file that an app could pull in.

**Comment by daveajones** [comment]
> It would be valuable to allow for multiple links to identifiying URIs, be they social media accounts, a Wikipedia page or a Podchaser Creator Profile - this would allow for the consuming platform to use whichever verification method they feel comfortable with to disambiguate people, as well as providing more options for linking out to useful pages for a specific person.
> 
> It's not quite as neat, but something like:
> 
> ```
> <podcast:person role="[host or guest]" img="[(uri of content)]">
>   <podcast:personName>[name of person]</podcast:personName>
>   <podcast:personUri>https://twitter.com/[username]</podcast:personUri>
>   <podcast:personUri>https://wikipedia.org/[page name]</podcast:personUri>
>   <podcast:personUri>https://podchaser.com/c/[Podchaser ID]</podcast/personUri>
> </podcast:person>
> ```
> 
> We could probably remove the `personXX` prefix as well if we're happy to rely on the context of the parent element to know what it relates to.

It's a good idea in principle.  But, man that adds a lot of complexity, considering there could be multiple persons with multiple links.  Seems like a lot of overhead.  I feel like apps would most prefer Podchaser and then have fallbacks.  Independent podcast apps would probably avoid IMDB since it's Amazon.

**Comment by douglaskastle** [comment]
Heaven forbid could there be a podcastIndex solution?  I like podchaser, but some day it might get bought out and i don't think I have had a single guest that has a podchaser profile.

At the start people search for a host or guest, they don't find it, they get a unique link, a plane one, like bit.ly.  As the ecosystem grows it turns out that one person is actually listed twice, with two links.  Well the system is setup so that the two people are merged into one, but the two links always resolve to that person.  One link is designate the primary one and is only given out going forward. There is probably some level of maintenance required in this though to resolve disputes.

The service could then be used to record the other links, imdb, twitter, podchaser etc

People are starting to use [linktree](https://linktr.ee/)  to sorta do this already, i don't know if one could suggest a standard way for a podcaster?

**Comment by daveajones** [comment]
Since there is still ongoing discussion here, I say we boot `<podcast:person>` to phase 2 and pick it back up after 11/15.  Everyone ok with that?

This is an important tag though, so I'd really like to have it and location be main focuses for phase 2.  I think we all knew these would be the most complex.

**Comment by jamescridland** [comment]
I think it's important to take our time on this, Dave, and I'd support moving this to phase 2.

@bslinger I think it would be important for us to, where we can, support the podcast taxonomy consortium's work where possible. Podjobs.net will certainly be doing so, and I know that Podchaser also has plans to do so. And I don't *think* it's too complicated to add to this spec to do this either.

Currently, we support "host" and "guest".

```
<podcast:person role="host" name="[name of person]" img="[(uri of content)]" href="[(uri to website/wiki/blog)]" />
<podcast:person role="guest" name="[name of person]" img="[(uri of content)]" href="[(uri to website/wiki/blog)]" />
```

How about

```
<podcast:person [group="cast"] role="host" name="[name of person]" img="[(uri of content)]" href="[(uri to website/wiki/blog)]" />
```

`Group` is optional. If it isn't present, it defaults to 'cast'.

For phase 2, we CamelCase-ify the taxonomy data, so that we use identical words to them. This allows this tag to be used for everyone working on the podcast, and identical data to Podchaser and others. This is a good thing.

So, my tags might be
```
<podcast:person role="host" name="James Cridland" href="https://james.cridland.net" />
<podcast:person group="audioPostProduction" role="composer" name="Chris Stevens" href="https://www.devaweb.co.uk" />
```

This has the benefit of aligning with the taxonomy work, which is important, as well as remaining entirely backwards-compatible with a simpler plan that just says "host".

Here's the full list, so they're visible without fiddling about with unzipping a CSV file:

```csv
Group Name,Position Titles,Description ,Example
Creative Direction,Director,"The Director is the head of the entire creative production, from creative details to logistics. There is typically a single director for a production. This role is primarily seen in fiction podcasts.","Jenna Knorr for ""Welcome to Tinsel Town"""
Creative Direction,Assistant Director,"The Assistant Director is a liason between the director and the rest of the production, often coordinating the daily logistics of production. There may be multiple assistant directors on a project. This role is primarily seen in fiction podcasts.","William Wright for ""Inn Between"""
Creative Direction,Executive Producer,"The Executive Producer is the lead producer on a production. The role can range in terms of creative control with some ""EP""s owning the creative direction of a podcast (in effect taking the role of director), while others may take a more hands off approach. Executive producer may have raised the money to fund the production, but it is not a necessary responsibility of the role.","Jane Rotonda for ""The Larry Meiller Show"
Creative Direction,Senior Producer,The Senior Producer is the second most senior producer of the production (second to the Executive Producer). They supervise producers and the general direciton and logistics of the entire production.,"Dr. Jeremy Weisz from ""INspired INsider"""
Creative Direction,Producer ,"The Producer coordinates and executes the production of the podcast. There duties can include helping craft the creative direction of a project, budgeting, research, scheduling, and overseeing editing and final production.",
Creative Direction,Associate Producer,The Associate Producer performs one or more producer functions as delegated to them by a Producer.,"Alex Baumhardt for ""APM Reports"""
Creative Direction,Development Producer,The Development Producer coordinates and executes the pre-production create direction of a podcast. Their responsibilities include finding new episode and series ideas and working with writers and researchers to prepare the concept for production.,
Creative Direction,Creative Director,"The Creative Director is responsible for the creative strategy and execution of an entire series. Often this role reaches outside of content to affect accompanying artwork, music, marketing campaigns, and more.","Neil Druckmann on ""The Official The Last of Us"""
Cast,Host,"The Host is the on-air master of ceremonies of the podcast and a consistent presence on every episode (with the exception of guest hosts and alternative episodes). The Host's duties may include conducting interviews, introducing stories and segments, narrating, and more. There may be more than one Host per podcast or episode.","Joe Rogan for ""The Joe Rogan Experience"""
Cast,Co-Host,"The Co-Host performs many of the same duties as the host, while taking a secondary presence on the podcast. ","Dax Shepard for ""Armchair Expert"""
Cast,Guest Host,"The Guest Host performs all of the duties of the traditional Host role, but does so in a temporary capacity. Often as a single appearance or a short span of episodes.","Erica Kelly on ""Let's Taco 'Bout Women and True Crime"""
Cast,Guest,"The Guest is an outside party who makes an on-air appearance on an episode, often as a participant in a panel or the interview subject. ","Lewis Brindley for ""Triforce!"""
Cast,Voice Actor,"The Voice Actor gives a performance in which they lend their voice to the role of a character on a podcast episode. While the majority of voice acting roles will be fictional, the role of voice actor may also cover reenactments of real conversations and people.","Venk Potula for ""Masala Jones"""
Cast,Narrator,"The Narrator gives a performance in which tell the exposition of a fictional or non-fictional story, often in a scripted manner. The Narrator may also perform voices of characters within the story, provided they still maintain the role of exposition storyteller or ""voice of God"".","James Harvey Freetly for ""Lakeshore & Limbo"""
Cast,Announcer,"The Announcer gives short vocal performances for the introduction of the podcast, episode topics, segments, guests, prizes, etc. The Announcer is secondary to the host of the podcast and often performs their introductions in a scripted, produced manner.","Lydia Kapp for ""World Builders Anonymous"""
Cast,Reporter,"The Reporter finds and investigates news or stories for the podcast, often interviewing subjects and conducting research. The Reporter can be an on-air position as well, as they convey the insights of their investigation.",
Writing,Author,The Author has written prose or poetry originally intended for text that is now being read verbatim on air.,"Heiko Martens for ""The Sigmund Freud Files"""
Writing,Editorial Director,"The Editorial Director heads all departments of the organization behind the podcast and is held accountable for delegating tasks to staff members and managing them. They are the highest-ranking editor and are responsible for the direction, accuracy, and decisions behind podcast content.","Christopher Twarowski for ""News Beat"""
Writing,Co-Writer,"The Co-Writer has written a podcast in partnership with 1-2 other writers, sharing credit together for the creative arc, dialogue, and narration.","Max Eggers on ""THE LIGHTHOUSE"""
Writing,Writer,"The Writer has written the story or dialogue of a podcast. The Writer is often involved in the creative arc of a production, but this is not a necessary requirement. Writers may work in fictional or non-fictional podcasts.",
Writing,Songwriter,The Songwriter has written the lyrics and/or accompanying music to an original song created for the podcast and played on an episode.,"Ben Lapidus for ""Gay Future"""
Writing,Guest Writer,"The Guest Writer performs the duties of a writer in a temporary capacity, often as a single episode or a short span of episodes. The distinction between writer and Guest Writer depends on the decision of the podcast itself.","Beth Crane for ""The Unseen Hour"""
Writing,Story Editor,The Story Editor is responsible for broad stroke direction of the story arc and character development of a podcast. Often seen in fiction and documentary podcasts.,"Gabrielle Loux for ""The NoSleep Podcast"""
Writing,Managing Editor,"The Managing Editor oversees and coordinates the podcasts editorial activities, providing both detailed editing and managing a staff of writers and editors to ensure proper deadlines and budgets are being met.","Flora Lichtman for ""Every Little Thing"""
Writing,Script Editor,"The Script Editor provides notes and editing to the recording script in a very ""hands on"" role. The Script Editor is primarily used in fiction, documentary, and advertisements where scripted recordings are prevalent.","Alex Rioux for ""Welcome to Tinsel Town: A Christmas Adventure"""
Writing,Script Coordinator,The Script Coordinator packages the final script with annotations that reflect specific logistics and creative cues for recording and production.,"Alex Rioux for ""Welcome to Tinsel Town: A Christmas Adventure"""
Writing,Researcher,"The Researcher coordinates the sourcing and verification of information that can then be used for the content of a podcast episode, often informing the direction of a story based on new insights uncovered.","Dave Grave for ""The Zero Brain Podcast"""
Writing,Editor ,"The Editor reviews and prepares scripts for conveying information in a creative, accurate, and engaging manner. ",
Writing,Fact Checker ,The Fact Checker reviews the content of a podcast for factual correctness and verifies that quote attribution is correct. They use a variety of tools including 3rd party research and individual outreach. Often the Fact Checker will also provide notes on how the production can avoid the confusion in the delivery of information in the episode.,
Writing,Translater ,"The Translator converts content from one language to another for the podcast. This can be interviews, dialogue, text documents, and more. The Translator's work may be used on-air or behind-the-scenes during the production/research process.",
Writing,Transcriber,"The Transcriber turns dialogue and audio cues into text, which can be used internally for production processes or displayed publicly for listeners.",
Writing,Logger ,The Logger reviews and documents the contents and timestamps of raw audio in service of producers and editors in the production process.,
Audio Production,Studio Coordinator ,The Studio Coordinator manages the recording studio and audio technicians working within the studio at the time of recording.,
Audio Production,Technical Director,"The Technical Director oversees the podcast's recording and production as it is involved with audio technologies including hardware and software, and managing roles involved these areas. ","Adam Raymonda on ""Celebuzz'd"""
Audio Production,Technical Manager,"The Technical Manager coordinates a team of audio engineers and studio staff, in the recording and production as it is involved with audio technologies including hardware and software.",
Audio Production,Audio Engineer,"The Audio Engineer helps record and produce audio by setting up recording environments, monitoring recoding, and providing technical adjustments throughout. The Audio Engineer is present during the recording process, most often making adjustments in real time. The Audio Engineer may work with conversation, music, foley, or any other type of audio.","Peter Leonard from ""Startup Podcast"""
Audio Production,Remote Recording Engineer ,"The Remote Recording Engineer ensures the proper recording of conversations taking place in multiple locations across a phone line or internet connection. The Remote Recording Engineer evaluates the different recording set ups and attempts to reconcile them into a cohesive sound, while also monitoring the recording process to capture the best possible audio.",
Audio Production,Post Production Engineer,The Post Production Engineer evaluates audio technologies and their application as it pertains to the final steps of production and publication.,"Dick Wound for ""Queens Next Door"""
Audio Post-Production,Audio Editor ,The Audio Editor cuts and rearranges audio for clarity and storytelling purposes. The Audio Editor may also perform general audio processing and mastering.,
Audio Post-Production,Sound Designer ,"The Sound Designer creates and composes a variety of audio elements. These elements are mostly secondary to speech, but a Sound Designer may creatively edit/produce speech elements in an artist manner.",
Audio Post-Production,Foley Artist ,"The Foley Artist sound effects for a podcast and can do so both via physical recording and digital processing, or a combination of the two.",
Audio Post-Production,Composer,The Composer writes an original musical piece (or multiple) that is played on the published episode. The Composer will also often be the performer of said musical piece.,"Marcus Thorne Bagala from ""This American Life"""
Audio Post-Production,Theme Music,"Theme Music is a musical piece that accompanies the podcast across multiple episodes, most often at the beginning of an episode. The Theme Music is used to introduce the podcast as a brand. This role is for the creator of the theme music.","Mark Philips from ""Startup Podcast"""
Audio Post-Production,Music Production,"The Music Production role helps creatively craft music in a role separate from the writing of said music. Music Production often involves creative decisions per the method in which music is recorded, the arrangement of instruments, the use of effects, and more.","Storm Duper for ""Faking Star Wars Radio"""
Audio Post-Production,Music Contributor,The Music Contributor is the creator of music that was used for the podcast but not necessarily produced specifically for the podcast. Often a podcast will use an existing musical piece and credit the original creator.,"Bobby Lord from ""Startup Podcast"""
Administration,Production Coordinator,"The Production Coordinator is responsible for managing the logistics of the production process from recording to publication, including attaining the required permissions and permits, connecting the various production and recording teams, coordinating the creation of post-production metadata, budgeting, and more.","Taneya Boyde on ""Ready For Change?"""
Administration,Booking Coordinator,"The Booking Coordinator is responsible for bringing on new guests for interviews, including sourcing guests, scheduling interviews, onboarding materials, and post-publication processes. ","Meryl Klemow for ""Campfire Sht Show"""
Administration,Production Assistant,"The Production Assistant helps support an executive member of a podcast (often a director or producer), helping prepare them in a variety of ways including scheduling, logistics, communications, and more.","Wallace Mack for ""The Nod"""
Administration,Content Manager,"The Content Manager is responsible for the distribution of a podcast's content within and outside of episode, including but not limited to clips, newsletters, images, cross-promotions, and more.","Kenneth Lee Johnson II for ""Malice Corp Smack Talk"""
Administration,Marketing Manager,"The Marketing Manager is responsibile for the promotion of a podcast's content through various awareness strategies such as social media campaigns, cultivating a web presence, managing public relations and communications strategies, and other creative techniques to acquire and retain listeners.",
Administration,Sales Representative,The Sales Representative is responsible for monetization of podcast content through managing and selling advertising inventory.,
Administration,Sales Manager,"The Sales Manager is responsible for all aspects of podcast monetization such as overseeing Sales Representatives, managing advertising inventory, and devising monetization strategies through channels such as affiliate partnerships, merchandise, live events, and other revenue strategies.",
Visuals,Graphic Designer,The Graphic Designer is someone who has created any custom visuals to accompany the podcast in a variety of ways.,"Sky Knight for ""The XP Billionaires"""
Visuals,Cover Art Designer ,"The Cover Art Designer creates the displayed cover art of a podcast or episode. For clarity, cover art is the main image (almost always square in dimensions) accompanying the podcast in directories, while episode cover art is displayed in a similar manner at the episode level. This role may be a digital designer, artist, photographer or any other visual creative.",
Community,Social Media Manager,"The Social Media Manager runs the social media accounts of the podcast, including but not limited to the creation of content, posting, replies, monitoring, and more.","Tom Joshi-Cale for ""World on a String"""
Misc.,Consultant,"A Consultant is a third-party position where someone from outside the organization works on a project, often offering a specific expertise. This is a modifier role and can be applied to any work area.","Ross Wilcock for ""Being Kenzie-Feature Length Immersive Horror"""
Misc.,Intern,An Intern is an apprentice position where someone works for a limited time within an organization to gain work experience in a specific field. This is a modifier role and can be applied to any work area.,
Video Production,Camera Operator,"A camera operator is responsible for capturing and recording all aspects of a scene for film and television. They must understand the technicalities of how to operate a camera, frame a proper shot with respect to lighting and staging, focus the lens and have a visual eye to achieve a specific look.",
Video Production,Lighting Designer,A lighting designer works with the DP and Director to craft a specific look and feel of a scene utilizing various lighting techniques. They must be able to interpret the creative direction and bring it to life.,
Video Production,Camera Grip,"A camera grip is responsible for building and maintaining all the parts of a camera and its accessories such as the tripods, cranes, dollies, etc.",
Video Production,Assistant Camera,"1st AC is responsible for the camera equipment, building the cameras before the start of each day, organizing all the parts and various accessories, swapping out lenses when necessary and also pulls focus for the DP and camera operators. The AC will also wrap out each day by cleaning the cameras, writing camera notes, marking the media cards, and delivering them to the DIT.",
Video Post-Production,Editor,Television editors are responsible for taking the shot footage and clips and blending them together to craft the director's vision and storytelling.,
Video Post-Production,Assistant Editor,"The Assistant Editor is responsible for taking the media from the set, ingesting them into the designated editing software, and organizing the footage in an efficient way for the editor. They must also pay close attention to ensure that audio and video are synced and that all footage from set is ingested properly.",
```

**Comment by jamescridland** [comment]
@douglaskastle @daveajones In terms of different URLs for people.

[There is a `rel="me"` specification](https://microformats.org/wiki/rel-me), which allows me to link, from my website, to other pages that I specify are about me on the internet. That could include Podchaser links and the like.

This allows one link to be placed in the RSS, and then the `rel="me"`'s to be spidered.

Further - if I were to `rel="me"` my podcast RSS feed from my website, and my podcast RSS feed links back to my website (either through `<link>` or `podcast:person`) then this appears to be a bi-directional link and thus helps considerably in proof that I really am that person in this podcast.

I don't know if it's entirely relevant here, but it does seem to fix the issue we're trying to fix.

**Comment by daveajones** [comment]
I like all of this James.  This is very thorough.

Is the Podcast Taxonomoy Consortium's work stable enough, at this point, to feel confident that it wouldn't change drastically out from under us structure-wise?  @bslinger It looks like Podchaser is heading that up?

**Comment by douglaskastle** [comment]
I suppose the important thing to achieve is to allow for the ability to separate two people with the same name from the same person being listed twice.  Also the goal, from my assessment, is that it is finding people as guests is the harder part, as they may be jumping around to podcasts is variable states of namespace support than if you are the host of your own, there is less churn in the host, in general.

The rel="me" look exactly like the start of the solution some where a person can maintain control.  It is fairly trivial to add it to ones own podcast, it might take a bit of professional dedication if one is a guest to get the podcast in question to put it up correctly, it is early days yet.  

However I am still trying to figure out how it is possible to improve the tag to enhance search.  Currently it seems that search on people name is a haphazard regex into the show title or show notes which i am sure results in a lot of false positives.  Maybe just getting the name in the tag it improves the data separation enough to make it practically useful again for search, who cares if it show two people with the same name, or the same person listed twice, as long as it is showing them.

**Comment by daveajones** [comment]
"Currently it seems that search on people name is a haphazard regex into the show title or show notes which i am sure results in a lot of false positives"  Yep, it's just a train wreck.  I really haven't even bothered trying for a guest search until we get something nailed down here because I just don't trust that the results will be compelling.

"The rel="me" look exactly like the start of the solution some where a person can maintain control. It is fairly trivial to add it to ones own podcast, it might take a bit of professional dedication if one is a guest to get the podcast in question to put it up correctly, it is early days yet." - The good part about this is that the guest themselves will probably be plenty motivated to make that happen and make sure it's right.  Nobody wants to guest on a show and then be misattributed.  Maybe this will be self-solving?  Seems like it will.

**Comment by douglaskastle** [comment]
Curious I found `rel=me` already implemented in the podlove plugin for the podcast I maintain, it seems to be used to point back to the apple itunes link for the same podcast.

	<atom:link rel="me" href="https://podcasts.apple.com/podcast/id1506675864" />

**Comment by daveajones** [comment]
That seems odd. I can’t find anything in the rel=“me” microformats spec to justify linking back to an apple podcasts page. I wonder if that’s just a sane default in case some other, more specific value isn’t entered.

**Comment by jamescridland** [comment]
The `rel="me"` specification is basically a deliberate link from one site to another to say "this is me". If you can link back - there will be a link back from Apple Podcasts to that website or feed - then that's vague proof that it really is you.

Google Podcasts actually uses similar - if you link to your homepage from your RSS feed, and your RSS feed from your homepage, then that offers a bit more certainty that it really is you. They don't use the `rel="me"` construct here though.

Another construction here is the schema.org [sameAs](https://schema.org/sameAs), which for Podnews looks like this:

```javascript
<script type="application/ld+json">
{
  "@context": "http://schema.org",
  "@type": "Organization",
  "name": "Podnews LLC",
  "url": "https://podnews.net",
  "logo": "https://podnews.net/f/favicon-180x180.png",
  "foundingDate": "2017",
  "founders": [
  {
  "@type": "Person",
  "name": "James Cridland",
  "givenName": "James",
  "familyName": "Cridland"
  }],
 	"contactPoint": [{
    "@type": "ContactPoint",
    "email": "updates@podnews.net",
		"url": "https://podnews.net",
    "contactType": "customer service"
  }],
  "sameAs": [
          "https://www.linkedin.com/company/podnews",
          "https://twitter.com/Podnews",
          "https://www.facebook.com/podnews"
        ]
}
</script>
```

Both of these are ways that we can suggest that *one* link is used per person, but that a podcast search engine may elect to spider the links used to help match individuals. We should recommend the use of `rel="me"` where possible.

Linktree, and similar, are perfect targets for this.

@bslinger I notice that https://www.podchaser.com/podcasts/podnews-podcasting-news-612989 doesn't `rel="me"` the social media or website links. You might want to consider doing that maybe.

**Comment by bslinger** [comment]
I understand the concerns about relying on Podchaser, but we (Podchaser) are putting in the work to solve a lot of these issues - the whole point of our creator database is to allow for disambiguation and more structured metadata about who is doing what on what show. Eg. on the subject of guest searching, we regularly search all of our episodes for the names of every creator in our database and have a team of moderators looking at the results and deciding whether it's actually an appearance by that person or just somebody mentioning them. A "PodcastIndex solution" would presumably require something similar to get similar results unless there is some additional innovation involved that we haven't thought of. As I mentioned earlier, I have no issues with making it optional, and if it's built in the right way it could be left open to any other service that wants to jump in and provide the ability to do some sort of identity verification.

@jamescridland , the `rel="me"` spec looks interesting, we can certainly think about adding that in - what are your thoughts on using it on the page representing a podcast vs the creator/user pages which represents a person?

**Comment by daveajones** [comment]
@bslinger I wouldn't want to build something on PI unless it was necessary (i.e. if Podchaser decided to exit that work).  My bigger concern is whether the taxonomy project is stable enough now to consider referencing it as James mentions.  Building on that naming convention seems like a good way to go, but only if it's not too early in the game.  I think we really want to get a guest/host tag out the door by the end of the year.  Do you know if the taxonomy project is at a point yet where we could start referencing it's cast nomenclature without risk of it changing out from under us?

**Comment by npruzaniec** [comment]
I agree here with the sentiment of keeping this simple but also wanting the ability for more roles. Feel like producer, donor, sponsor, cast roles are all very valuable even if not surfaced in most players. As you expand that, there's even more of a need for a `rel="me"` style relationship for any credit given.

**Comment by daveajones** [comment]
At this point can the tag proposal change just include a reference to the taxonomy project roles list as the standard, and a link back to any page for the person in the spirit of `rel="me"`.

Is that enough to get us back on track?  If so, I can write it up and we can use that as a new starting point.

**Comment by jamescridland** [comment]
> My bigger concern is whether the taxonomy project is stable enough now to consider referencing it as James mentions.

Stability - are you looking for stability in the mechanism of referring to a position by using the equivalent of a "Group Name" and "Position Title" (what we're calling `group` and `role`? Or are you looking at whether the *values* of those property names are stable?

Cole Raven of Podchaser, who works with @bslinger , appears to be the driving force behind the taxonomy project. Since August, I've been looking into it from afar, though missed the last group call. I hope that Ben might be able to raise his awareness of this thread - I have put something into the Slack group a while back. I shall also mail Cole directly and hope he takes part here.

Without speaking for Cole, my understanding is that the concept of a "Group Name" and "Position Title" is stable (it's always been part of the documents shared around). I'd also guess that the actual values of those are, given the nascent industry that podcasting is in, not ever going to be 'stable'.

The "official first draft" is due on November 20th; and Cole, in his email, says:

> By mid-December, we'll plan an announcement date to share the new Podcast Taxonomy Consortium website and first draft, encouraging all publishers, service providers, etc to support this initiative and join the conversation.

My suggestion is that Podcast Index recommends, at the very least, `role="host"` and `role-="guest"`; but also highlights that this specification is ready for integration with the Podcast Taxonomy Consortium; and that the optional `group` parameter defaults, if not present, to "cast".

To that end, if the requirement for "stability" is a requirement that 'host', 'guest' and 'cast' are immutable and stable, I think that would be achievable.

**Comment by daveajones** [comment]
Excellent. That answers my questions. I think we can definitely do that. Podcastindex will definitely support the taxonomy projects work.  I’ll make the change to the tag as you’ve outlined and we can use that as a base to move forward. If anything changes when the draft is released that would modify that we can address it then.

Thank you @jamescridland

**Comment by brianoflondon** [comment]
I'd like to see the roles mentioned here also reflected in value tags for sharing value.

**Comment by douglaskastle** [comment]
I would just like to add, and this I think was touched on in the recent podcasting 2.0 podcast and also some of the conversation over on the archive tag, about how some have blown up in size.  The goal should be to keep the RSS feed a trim and light as possible.  Sure with this tag it would be possible to do a full production cast listing, but that seems too heavy. If you want full cast listing, put it in the show notes.

For all tags, not just this one, what is the goal.  

I think the reality of what this tag is trying to achieve enabling search for guests across podcasts, i don't think there is much demand for tracking producers or interns (but i could be wrong the internet never seems to throw up surprises).

However this is just an observation. Podcasters can do what ever they want, but if they are looking for some recommended guidelines i think it should hew to the more trim version, in the RSS feed.

**Comment by daveajones** [comment]
I've updated this tag to reflect the most current understanding after talking it over with @bslinger .  I think this is something we can work with to move forward and tweak as necessary.

**Comment by daveajones** [comment]
@douglaskastle 99% of the time it's just going to be host, co-host, guest specified.  But, linking to the fuller cast list allows bigger productions to come to the party as well.  Those productions may have union requirements and such that they could satisfy with fuller credits.  I think that will be rare.  But, making it possible, for little cost is a good idea.  Simplicity for the masses with flexibility for the few.

**Comment by Majester** [comment]
Please consider Narrator and Performer/Artist for locking in or easy assignment. Podcasts and podplayers get used for audio books and audio dramas,  some host music and others embedded music needing easier attribute.

**Comment by jamescridland** [comment]
As of today, the Podcast Taxonomy Consortium has published their white paper -  https://podnews.net/update/taxonomy-consortium links there - and I plan to make podjobs.net support this list in full as soon as I can.

One of the pre-requisites of me doing that is for their big list to be in a JSON format file somewhere in a format that's better than a damn PDF! I'm hoping that I can convince someone to put it in a Github repo somewhere, though have a concern that it's likely that the "someone" will be me...

**Comment by daveajones** [comment]
@jamescridland Did the roles lost change significantly?  I have the CSV file. I could write a script to convert it to JSON if you post the structure you’d like to see. I can add it to the repo here which needs to be done anyway.

**Comment by daveajones** [comment]
I guess I should have asked @bslinger that.

**Comment by daveajones** [comment]
> One of the pre-requisites of me doing that is for their big list to be in a JSON format file somewhere in a format that's better than a damn PDF!

Done!  JSON file is [in the repo](https://github.com/Podcastindex-org/podcast-namespace/blob/main/taxonomy.json) now.  Tell me if you need it to look different.

I also referenced it in the tag documentation [here](https://github.com/Podcastindex-org/podcast-namespace#phase-2-open).

**Comment by daveajones** [comment]
I'm think this tag is done. Since the taxonomy project was the last thing we were waiting on, I move to finalize it now.

**Comment by daveajones** [comment]
@jamescridland What say you?

**Comment by jamescridland** [comment]
@daveajones Looks good to me!

**Comment by daveajones** [comment]
@jamescridland If we do it in one place, we should also make the change here so that the name is displayed as the node value, like this:  https://github.com/Podcastindex-org/podcast-namespace/issues/138#issuecomment-753698941

**Comment by stuartjmoore** [comment]
[New here, jumping in late, apologies]

Why do we need a `role` and `group` taxonomy? That seems like a lot of overhead for what is effectively a human-readable string. It would be impossible to cover all the cases, just off the top of my head I can think of podcasts with “sidekick”s, “drop-guy”s, “news person”s that may not be covered by the list.

If any calculation is needed, `role` could function as an identifier unique to that podcast or episode.

**Comment by jamescridland** [comment]
Hi, @stuartjmoore - the role/group is specifically set so that this *is* machine-readable as well as humanly-readable. It's from [The Podcast Taxonomy](https://podcasttaxonomy.com/) group - who would be very happy to learn of any omissions. I'd have sidekicks/dropguys as a co-host, personally.

**Comment by stuartjmoore** [comment]
Is there a reason to make it machine readable? I worry it won’t scale. And maintaining a taxonomy list sounds impossible.

**Comment by jamescridland** [comment]
"Show me all executive producers of podcasts in Turkish"

"Show me any podcast host who's done a show about Disneyland"

> maintaining a taxonomy list sounds impossible.

Thankfully, we don't need to worry about that. That's what the Podcast Taxonomy group is for: all we need to do is follow their lead.

**Comment by daveajones** [comment]
@stuartjmoore Thanks for getting involved Stuart. The way the tag is written is to have sane defaults for role and group so that 90+% of the time those attributes aren’t even needed. It’ll be the rare podcast that wants to specify “audio engineer” in their “person” tag. But if they do, they can. But if you just want “host” or “guest” it’s dead simple. We tried to write it so that it can scale up and down in complexity.

**Comment by stuartjmoore** [comment]
I’m loving everything, so I hope I don’t sound like a nay-sayer. I’ve even tried to start my own podcasting standard more than once dating back to 2009.

My general goal when engineering is to build it up complexly, then tear it back down to simplicity, removing redundancy, and opening it up to scale. I don’t think the taxonomy is a bad idea; I was more curious if there was a goal for the added complexity.

If the field is open to custom strings—and not restricted to an enumeration—it fits both our goals 👍🏻

**Comment by PofMagicfingers** [comment]
I know I'm a bit late, given this is already Phase 2, however I'd like to share with you what we worked on. 

I guess it could be difficult to add it later in the spec without compatibilities issues, because of the name of the person being the content of the tag.

On our parallel work on [podCloud/podcast-ext](https://github.com/podCloud/podcast-ext) (soon dead and replaced by podcast-namespace) we intended to allow `social` and `link` tags as `person` children. That would be great to add more than one link for one person.

That would let us do something like this.

```xml
<podcast:person role="host" group="cast" img="[uri]" href="[maybe some "main" link]" name="Pof Magicfingers">
	  <podcast:social platform="twitter" url="https://twitter.com/pofmagicfingers">@PofMagicfingers</podcast:social>
	  <podcast:social platform="github" url="https://github.com/pofmagicfingers"/>
	  <!-- I'll do a proposal for this generic link tag on another issue -->
	  <podcast:link href="https://jaimelegif.com">Thomas project website</podcast:link>
</podcast:person>
```

Also in our spec we planned to support a more precise "identity" for the person, including firstname, lastname, nickname, and pronoun (from a community fixed list). Those would be optional but could be a good thing when implemented by clients.

```xml
<podcast:person firstname="Giovanni" lastname="Olivera" nickname="PofMagicfingers" pronoun="he" />
```

Let me now what you think of these ideas, and if we could/should salvage them merging them on this tag proposal, or maybe in some other way.

**Comment by stuartjmoore** [comment]
I like `link` tags being children of `person` tags, because it gives `person`, `item` (episode), and `channel` (show) all the same semantics. A `link` can exist in a `channel`, `item`, or `person`. As well as a `person` can exist in a `channel` or `item`.

Given that shows, episodes, and people are all nouns, it make sense for them all to act similarly and share attributes. `image` would be another that is defined externally, but placed in any of the three.

**Comment by PofMagicfingers** [comment]
> I like `link` tags being children of `person` tags

Hey that could be cool, I think this is technically possible to allow original tags to be children of ours.

**Comment by daveajones** [comment]
One thing we've consistently heard from hosting companies is that they are worried about tag bloat in the feed.  Every extra bit of tag ends up adding bandwidth  for a host that has tens of thousands of feeds.  So, with each step we've tried to keep it as lean as possible.

One of the things that caused us to move the name value into the node instead of an attribute was the need for displaying the value in an XSL style template, which is very common among hosting companies.

This tag (and the location tag) took a ton of work and arguing to get to the point they're at now, so I'm hesitant to mess with it.  These are good ideas though. So, I think we can revisit this for an expansion/enhancement after we see how things go in the real world.  Part of this project from the beginning has been the idea that we would bolt on new features to tags in future releases.  So, I think that's the best way to handle.

**Comment by PofMagicfingers** [comment]
Yeah sure, no worries. We also are a hosting company and have thousands of feeds, so I'm all ear for these arguments.

I stilled had to brought up our thoughs on this tag "just in case", but I'm pretty happy with your version. 
It's indeed less bloated, and even if we tried to be the most exhaustive in our original ideas, I'm not sure we really need social platforms and websites for persons: one link should be enough.

PS : You actually can [use attributes in XSL](https://www.xml.com/pub/a/2002/04/03/attributes.html), but I still also prefer the name as text content for simplicity.

**Comment by swschilke** [comment]
Just came across this at Apple https://help.apple.com/itc/podcastshostandguestimageguidelines/en.lproj/static.html and https://itunespartner.apple.com/podcasts/articles/hosts-and-guests-3085

I'm wondering if in the person tag everything is "there" so that it would work with this description?

.

**Comment by PofMagicfingers** [comment]
I think this document works for special podcast pages with special
agreements with Apple. I'm pretty sure there is no official way in the rss
feed to have your hosts and guests info in Apple Podcast, but I could be
wrong.

Anyhow, it can be interesting to keep that content in mind when documenting
our tag picture guidelines, I guess.

**Comment by jamescridland** [comment]
I'm in Apple Podcasts as a 'person'. You need to know the right people and ask the right questions - you then email them with an image which needs to have a specific naming convention.

You probably also need to invite your friends into a circle, turn off all the lights, have eight candles, and chant your allegiance to the one true visionary Steve Jobs fifteen times, and then successfully log into Podcasts Connect. Most people fail at the last hurdle.

(It's a proprietary thing for Apple only. But no harm in stealing their spec in terms of image sizes.)

**Comment by johnchidgey** [comment]
The loose end I have is the current lack of an option to tag/flag a "Location" for a podcast "Person". As I see it, I may want to search on podcasts created by those living in certain places. Something like <podcast:personLocation> or similar. It would replicate the already agreed <podcast:location> formatting so there's nothing else to add/modify in that regard.
Of course it would be better to have that embedded in the non-Podchaser future-state format if/once that can be agreed and then the podcast app/index can subsequently scrape this and it's the person/owner of their own identity to set that rather than the RSS feed owner, but that requires the cart before the horse in a manner of speaking.

**Comment by jamescridland** [comment]
"Location" is not for 'created in a certain place', or even 'podcast from a certain place', it is for 'editorial about a certain place'.

Unless we wish to widen the scope of the location tag, I wouldn't support adding a location tag to a person element. The location tag can be used for a specific show where that show is about a place ("The Fortitude Valley Podcast") or a thing ("The Brisbane Wheel") but if it's a show about podcasting news, for example, the fact it comes from Bardon QLD is not the intended use for the location tag.

The original spec for the location tag had a rel="from" or rel="about" tag, but it was quite strongly argued against for what must have been good reasons at the time.

**Comment by johnchidgey** [comment]
There are two debates to be had: 1) should there be a way to add the location of a creator to their personal profile in a searchable way; 2) whether the intent of the existing location tag should change. My suggestion isn't the latter, it's the former. I'm open to a discussion over whether that adds value or not as it relates to the person tag. I'm curious if anyone thinks this information has value.

**Comment by jamescridland** [comment]
I guess my concern is: "location is intended to communicate editorial about a certain place, unless its within a person tag, in which case location is that person's home city" is a little messy.

I've focused on the past on "hear all podcasts about breweries in Queensland", which is where the `location` tag should excel (as long as you use the OSM portion). Or "hear all podcasts about Enoggera".

Is there a use-case in "hear all podcasts made by people who live in Logan City"? I'm not sure, but could be convinced.

**Comment by swschilke** [comment]
I think there should be the possibility to store the following locations, each in different degrees of detail (Country, Region, City, ...).

1. - The Podcast (from where)
2. - the podcaster (if wanted)
3. - the topic(s) (about "where") 
4. - the participant(s) (if wanted)

 
Not everything fits into the location tag (that, in my eyes, would be the first and third), but the person related  location information should be tied to the person record.

OSM is in my eyes tied to much to Open Streetmap, there are location based ways to record a position or area (based on standards, like lat/long).

Can we agree on this?

.

**Comment by jamescridland** [comment]
Sure, multiple tags rel="from" or rel="about" (i.e. "I'm in Brisbane, but this podcast is about Sydney") is absolutely fine from a tech point of view. I'd question how bloated we want to make the `person` tag, and of the relevance of the physical location of where someone is sitting. We appear to have re-opened the `location` tag, which is set: it's purely to describe the editorial focus of the podcast, and not to describe where a podcast is recorded or edited.

Also for a person - surely this, along with lots of other data, could be programmatically divined from the suggested Podchaser link, or Twitter link, or similar? If someone is from New York, should this data be available once, or in two hundred different podcast RSS feeds?

>OSM is in my eyes tied to much to Open Streetmap, there are location based ways to record a position or area (based on standards, like lat/long).

OSM *is* a standard, and the APIs to OpenStreetMap are open, and the data is available free with attribution. It's always used in the location tag as an *addition* to the geo tag, which includes lat/lon (and altitude, and a geospace like "Earth"). OSM allows you to say "This is about the Eiffel Tower in Paris" (which is a tourist attraction, and a tower, and in this arrondisement, and is called Tour d'Eiffel in French) rather than "this is about 2.38734234,12.3826323" which only refers to a specific point on the planet. If we don't have the option of using OSM here, we're reduced to specific points on the planet rather than the ability to describe places properly. That would be disappointing, since it kills the potential of "podcasts about tourist attractions in France" and instead only allows you to search for "podcasts about places in Paris" which is much less rich.

I'm proud that the `location` tag uses `geo` as a requirement, but `osm` as a recommended addition. It allows us to do some amazing things in future. Let's not try to kill that future flexibility here.

Anyway - I think I'll bow out of this; but would really rather we don't a) reopen the `location` tag, or b) try to shoehorn location details about people into the RSS feed. I don't see what benefit it offers, and I can see plenty of downside.

**Comment by swschilke** [comment]
Just wanted to mention https://schema.org/Place and https://schema.org/GeospatialGeometry which could be a model we could use. I think OSM ist not a "real" standard.

.

**Comment by johnchidgey** [comment]
>
>
> Also for a person - surely this, along with lots of other data, could be programmatically divined from the suggested Podchaser link, or Twitter link, or similar? If someone is from New York, should this data be available once, or in two hundred different podcast RSS feeds?
> 
>
James, from my original post/note I stated: "...it would be better to have that embedded in the non-Podchaser future-state format..." for which I agree - I don't think it should be replicated. Therefore in the interests of solving a potential issue, should a non-Podchaser format be the next thing to consider, of which the podcasters location should belong.

And if that's the case, then it's only tangentially related to the podcast:person tag insofar as we reference that identity link rather than podchaser.

**Comment by picard102** [comment]
Couldn't find if this was addressed anywhere, but what happens with persons with multiple roles? 

Do we output them multiple times for each role (not ideal), or do we comma separate the roles in the role tag?

**Comment by daveajones** [comment]
Right now it would be multiple outputs. We can always revise in a future spec to allow for comma separated roles.

**Comment by Marzal** [comment]
Implemented by https://github.com/Podcastindex-org/podcast-namespace/pull/29 and https://github.com/Podcastindex-org/podcast-namespace/pull/390

Docs:
- https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/tags/person.md - [permalink](https://github.com/Podcastindex-org/podcast-namespace/blob/336978220c14c0f180282ee0feb3e027567c4522/docs/tags/person.md)

Feel free to open a new [discussion](https://github.com/Podcastindex-org/podcast-namespace/discussions/categories/enhancement-proposal) for a request for changes aka `enhancement-proposal`
